cirq-core/cirq/devices/grid_device_metadata.py

```diff
@@ -139,7 +139,19 @@ def compilation_target_gatesets(self) -> Tuple['cirq.CompilationTargetGateset',
 
     @property
     def gate_durations(self) -> Optional[Dict['cirq.GateFamily', 'cirq.Duration']]:
-        """Get a dictionary mapping from gateset to duration for gates."""
+        """Get a dictionary mapping from gateset to duration for gates.
+
+        To look up the duration of a specific gate instance / gate type / operation which is part of
+        the device's gateset, you can search for its corresponding GateFamily. For example:
+
+        >>> my_op = cirq.Z
+        >>> gate_durations = grid_device_metadata.gate_durations
+        >>> op_duration = None
+        >>> for gate_family in gate_durations:
+                if my_op in gate_family:
+                    op_duration = gate_durations[gate_family]
```

Comment:
Need to put the `...` on these lines (just as it would appear in the python interpreter.